src/qamqpchannel.cpp

```diff
@@ -210,14 +219,19 @@ void QAmqpChannelPrivate::closeOk(const QAmqpMethodFrame &)
     Q_Q(QAmqpChannel);
     Q_EMIT q->closed();
     q->channelClosed();
-    opened = false;
+    newState(ChannelClosedState);
 }
 
 void QAmqpChannelPrivate::openOk(const QAmqpMethodFrame &)
 {
     Q_Q(QAmqpChannel);
     qAmqpDebug(">> OpenOK");
-    opened = true;
+
+    newState(ChannelOpenState);
+    if (qosDefined)
+        q->qos(requestedPrefetchCount,
```

Comment:
On 09/04/15 22:40, Matt Broadstone wrote:
> http://wiki.qt.io/Qt_Coding_Style#Line_breaks, we have 100 columns to
> work with so I think this fits on a single line

Yep, fair point.  I normally work with 78 columns for historic reasons,
also I find I can fit two editor windows side-by-side then on a 1366×768
LCD screen and not have any line wrap.

0b797c318c086c4d3574b258101bd7f66f23d198 corrects this, and I'll have a
look at some of the other files too.
-- 
Stuart Longland (aka Redhatter, VK4MSL)

I haven't lost my mind...
  ...it's backed up on a tape somewhere.